### PR TITLE
Factorise version- and set- controllers into the book one

### PR DESCRIPTION
**Pull request by Niols**
This PR abandons the set controller to make it rely entirely on the book one. This reduces code size and increases consistency. This ensures that both sets and books enjoy future improvements at the same time trivially. Ideally, we'd also do that for versions, but that is its own entire mess. I will at least do it for version PDFs though.

### CODE REVIEW

**Comment by Niols** [comment]
Now also done for version PDFs.

File: src/server/dancelor_server.ml [hunk 1]
```diff
@@ -108,9 +111,7 @@ let initialise_logs () =
 
 let populate_caches () =
   Dancelor_server_controller.Version.Svg.populate_cache ();%lwt
-  Dancelor_server_controller.Version.Pdf.populate_cache ();%lwt
```
Comment (R1kM) [inline src/server/dancelor_server.ml:111]: I'm slightly surprised that the version pdf cache is also removed, but I might be missing something

**Review by R1kM — APPROVED** [review]
Looks good to me, thanks!

File: src/server/dancelor_server.ml (hunk 1 above)
Comment (Niols) [reply src/server/dancelor_server.ml:111]: Because I said “Now also done for version PDFs.”!

Comment (Niols) [reply src/server/dancelor_server.ml:111]: But I guess this warrants renaming the PR.

**Pull request merged**